plugins/modules/datahub_definition_info.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cloudera.cloud.plugins.module_utils.cdp_common import CdpModule
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: datahub_definition_info
+short_description: Gather information about CDP Datahub Cluster Definitions
+description:
+    - Gather information about CDP Datahub Cluster Definitions
+author:
+  - "Webster Mudge (@wmudge)"
+  - "Dan Chaffelson (@chaffelson)"
+requirements:
+  - cdpy
+options:
+  name:
+    description:
+      - If a name is provided, that Definition will be described.
```

Comment:
Both `name` and `crn` are supported. Docs and `aliases` should be updated to reflect.